microshift_running_apps/microshift-operators.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id="operators-with-microshift"]
+= Using Operators
+include::_attributes/attributes-microshift.adoc[]
+:context: operators-microshift
+
+toc::[]
+
+Operators integrate with Kubernetes APIs and CLI tools such as `kubectl` and `oc`. Operators provide the means to manage applications and their resources, such as deploying a database or message bus. The Operator Lifecycle Manager (OLM) is not supported by {product-title}, but you can install Operators with manifests.
```

Comment:
```suggestion
Operators integrate with Kubernetes APIs and CLI tools such as `kubectl` and `oc`. Operators provide the means to manage applications and their resources, such as deploying a database or message bus. The Operator Lifecycle Manager (OLM) is not included with {product-title}, but you can install Operators with manifests.
```

OLM works on MicroShift, we just don't include it or expect users to want to use it.

If you want to get into the reasoning and focus more on the "what works" instead of what does not work, we could say something like "In order to minimize the footprint of {product-title}, Operators are installed directly with manifests instead of through the Operator Lifecycle Manager (OLM)."